src/utils/OpenSwathDIAPreScoring.cpp

```diff
@@ -145,7 +147,8 @@ class DIAPreScoring :
       std::cout << ltrans << std::endl;
     }
     // Here we deal with SWATH files (can be multiple files)
-
+    DIAScoring dia;
+    auto isotope_cache = dia.getCache();
```

Comment:
Could you not simply instanciate an `IsotopeDistributionCache isotope_cache;` here directly?
(remove the DIAScoring.h in this case)